src/modules/licence-import/mappers/company-contact.js

```diff
@@ -0,0 +1,21 @@
+const date = require('./date');
+
+const mapCompanyContacts = (contact, licenceVersions) => {
+  if (contact === null) {
+    return [];
+  }
+
+  // Ignore draft versions
+  const filtered = licenceVersions.filter(row => row.STATUS !== 'DRAFT');
```

Comment:
Hi James, same thing as before - not sure if you have to filter out the drafts here?